server/controllers/shoppingList.js

```diff
@@ -236,10 +189,9 @@ const updateShoppingListItem = (req, resp) => {
    * are passed in the request
    *  */
   const propertiesToUpdate = {};
-  typeof isOrdered !== 'undefined'
-    ? (propertiesToUpdate['products.$.isOrdered'] = isOrdered)
-    : null;
-  voterIds ? (propertiesToUpdate['products.$.voterIds'] = voterIds) : null;
+  typeof isOrdered !== 'undefined' &&
+    (propertiesToUpdate['products.$.isOrdered'] = isOrdered);
+  voterIds && (propertiesToUpdate['products.$.voterIds'] = voterIds);
```

Comment:
Please do not use `&&` in this way - use simple `ifs`.
That kind of `&&` operator usage is ok for jsx.